content/3_Silver/Func_Graphs.mdx

```diff
@@ -665,6 +802,38 @@ public class badge2 {
 
 </JavaSection>
 
+<PySection>
+
+```py
+import sys
+
+# may need for large inputs
+sys.setrecursionlimit(1000000)
+
+vis = [0] * MAXN
```

Comment:
i don't see maxn defined anywhere

Reply:
yeah it's sample code -- the c++ solution does something similar

Reply:
that seems like an issue
i'll remind myself to do a maintenance pr sometime